env/production/aws-auth-configmap.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ConfigMap
+metadata:
+  name: aws-auth
+  namespace: kube-system
+apiVersion: v1
+data:
+  mapRoles: |
+    - groups:
+      - system:bootstrappers
+      - system:nodes
+      rolearn: arn:aws:iam::296255494825:role/eks-worker-role
+      username: system:node:{{EC2PrivateDNSName}}
+    - groups:
+      - system:masters
+      rolearn: arn:aws:iam::296255494825:role/AWSReservedSSO_AWSAdministratorAccess_dcf2167fdeb47617
+      username: AWSAdministratorAccess:{{SessionName}}
+    - rolearn: arn:aws:iam::296255494825:role/notification-admin-apply
+      username: notification-admin-apply
+      groups:
+        - system:masters
```

Comment:
I don't think we need this change in prod... the admin repo only touches the staging k8s cluster, we do all the prod k8s stuff through the manifest repo.

Reply:
We do need this role ARN added instead though:
```
arn:aws:iam::296255494825:role/notification-manifests-apply
```